### PR TITLE
Feature/beta calculation aid

### PR DESCRIPTION
**Pull request by kzalite**

### CODE REVIEW

**Comment by kzalite** [comment]
tests pass! quickly merge!

File: app/ts/ManualInput/CellValidityService/CellValidityService.ts [hunk 1]
```diff
@@ -12,41 +12,65 @@ const NUMERIC_INPUT_ERROR = 'Please provide a numeric input';
...
-  } else if (isBetaValueInvalid(alpha)) {
+  } else if (isNotPositiveInteger(alpha)) {
     return 'Alpha must be an integer above 0';
   } else {
     return '';
   }
 }
 
-function isBetaValueInvalid(value: number): boolean {
+export function getBetaBetaError(alpha: number): string {
```
Comment (DanielReid) [inline app/ts/ManualInput/CellValidityService/CellValidityService.ts:22]: betabeta?

Comment (DanielReid) [reply app/ts/ManualInput/CellValidityService/CellValidityService.ts:22]: ohh, beta param beta distribution. still though :P

File: test/endToEnd/manualInputTest.js [hunk 2]
```diff
@@ -720,14 +723,30 @@ function enterDistributions(browser) {
     .pause(seleniumDelay)
     .click(typeSelector)
     .click(beta)
+    //FIXME
```
Comment (DanielReid) [inline test/endToEnd/manualInputTest.js:726]: ?

**Review by DanielReid — APPROVED** [review]
fixme?

**Review by chinook25 — APPROVED** [review]

**Pull request merged**